Bond/Bond+UIButton.swift

```diff
@@ -27,13 +27,14 @@
 
 import UIKit
 
-@objc class ButtonDynamicHelper
+class ButtonDynamicHelper: NSObject
```

Comment:
Let's not worry about naming right now. I'm working on next major version of Bond that will have all names fixed. I might need some input in few days. Keep you posted....